Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -347,14 +347,23 @@ static void DTKStreamingCallback(DIInterruptType interrupt_type, const std::vect
   }
 }
 
+static bool ShouldLidBeOpen()
+{
+  // Disc Channel relies on cover being open when no disc is inserted
+  // The Wii also has no physical cover
```

Comment:
About that... 

![wiimini](https://user-images.githubusercontent.com/6551020/113938718-d398a780-97af-11eb-9d3b-cf4da86d9dc0.jpg)

Of course we don't target the Wii Mini, so this is all semantics. Might be nice to include a nod to this in the comment though!

Anyway, I'll take a look at the UI for this shortly and do a follow up.